cl/phase1/network/gossip_manager.go

```diff
@@ -131,7 +131,7 @@ func (g *GossipManager) onRecv(ctx context.Context, data *sentinel.GossipData, l
 	if errors.Is(err, services.ErrIgnore) || errors.Is(err, synced_data.ErrNotSynced) {
 		return nil
 	}
-	if err != nil {
```

Comment:
not sure what to do in these cases...so keeping existing behavior. cc: @Giulio2002